src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/ConfigCommands/ConfigRunners.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Globalization;
+using System.IO;
+using NuGet.Commands;
+using NuGet.Common;
+using NuGet.Configuration;
+
+namespace NuGet.CommandLine.XPlat
+{
+    internal static class ConfigPathsRunner
+    {
+        public static void Run(ConfigPathsArgs args, Func<ILogger> getLogger)
+        {
+            var settings = RunnerHelper.GetSettingsFromDirectory(args.WorkingDirectory);
+            ILogger logger = getLogger();
+
+            var filePaths = settings.GetConfigFilePaths();
+            foreach (var filePath in filePaths)
+            {
+                logger.LogMinimal(filePath);
+            }
+        }
+    }
+
+    internal static class RunnerHelper
```

Comment:
you don't need a separate class for 1 static method when you're already in a static class :)

I'd just merge these 2 in 1.

Reply:
We're trying to follow what other commands look like, so our plan is to add in additional classes for the other `dotnet nuget config` commands to this file. Having `RunnerHelper` be a separate class will be more meaningful once we implement more commands.

Reply:
I guess, it's just hard to see that without more changes :) 

I'll let that be your call (whether you change it now), and I can provide feedback on it when I see the rest of the code. Feel free to resolve the thread once you've decided what you want to do.